ArduPlane/servos.cpp

```diff
@@ -202,7 +202,8 @@ void Plane::dspoiler_update(void)
     const int8_t bitmask = g2.crow_flap_options.get();
     const bool flying_wing       = (bitmask & CrowFlapOptions::FLYINGWING) != 0;
     const bool full_span_aileron = (bitmask & CrowFlapOptions::FULLSPAN) != 0;
-    const bool progresive_crow   = (bitmask & CrowFlapOptions::PROGRESSIVE_CROW) != 0;
+    //progressive crow when option is set or RC switch is set to progressive 
+    const bool progresive_crow   = ((bitmask & CrowFlapOptions::PROGRESSIVE_CROW) != 0  || crow_mode == Plane::Crow_Mode::Progressive); 
```

Comment:
I left out a bracket, ion my suggestion, I meant this ((bitmask & CrowFlapOptions::PROGRESSIVE_CROW) != 0)  || crow_mode == Plane::Crow_Mode::Progressive;

Reply:
if this is correct style I can change....it is correct in terms of execution, right?

thanks for reveiwing...

Reply:
It worked fine, so I guess it makes no difference, but 
((bitmask & CrowFlapOptions::PROGRESSIVE_CROW) != 0)  || crow_mode == Plane::Crow_Mode::Progressive
(bitmask & CrowFlapOptions::PROGRESSIVE_CROW) != (0  || crow_mode == Plane::Crow_Mode::Progressive)
are not the same, in this case I guess it comes out as the first, not being a expert in logic ordering I tend to put the the brackets, I think it also makes it clearer to read. As you have it now its just a extra set of brackets round the whole statement.

Reply:
changed